openpype/hosts/maya/api/workfile_template_builder.py

```diff
@@ -234,26 +234,10 @@ def get_placeholder_options(self, options=None):
         return self.get_load_plugin_options(options)
 
     def cleanup_placeholder(self, placeholder, failed):
-        """Hide placeholder, parent them to root
-        add them to placeholder set and register placeholder's parent
-        to keep placeholder info available for future use
+        """Hide placeholder, add them to placeholder set
         """
-
         node = placeholder._scene_identifier
-        node_parent = placeholder.data["parent"]
-        if node_parent:
-            cmds.setAttr(node + ".parent", node_parent, type="string")
-
-        if cmds.getAttr(node + ".index") < 0:
-            cmds.setAttr(node + ".index", placeholder.data["index"])
-
-        holding_sets = cmds.listSets(object=node)
-        if holding_sets:
-            for set in holding_sets:
-                cmds.sets(node, remove=set)
 
-        if cmds.listRelatives(node, p=True):
-            node = cmds.parent(node, world=True)[0]
```

Comment:
- It is not useful because it moves the placeholer into the root hierarchy, while it is hidden in the scene and in outliner.
- And this is problematic because the identifier attribute of the placeholer is the fullpath of the placeholder, so if we move the placeholder it is considered another placeholder.

Reply:
Sounds reasonable to me, @BigRoy ?

Reply:
Reasoning sounds fine, yes. But it's hard to tell for me what production use cases this influences, I haven't used this functionality at all.